packages/api-specification/src/hiddenWindow.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
I wonder if _mainWindow_, _masterWindow_, _hiddenMainWindow_ or something like that would be a better name? To indicate to the user the this isn't just a normal window which is hidden, but something different?